lib/byron/src/Cardano/Wallet/Byron.hs

```diff
@@ -279,6 +297,86 @@ instance DefineSeverity ApplicationLog where
         MsgDatabaseStartup ev -> defineSeverity ev
         MsgServerStartupError _ -> Alert
 
+instance ToText IPVersion where
+    toText IPv4 = "IPv4"
+    toText IPv6 = "IPv6"
+
+instance ToText NtpTrace where
+    toText msg = case msg of
+        NtpTraceStartNtpClient ->
+            "Starting ntp client"
+        NtpTraceTriggerUpdate ->
+            "ntp client is triggered"
+        NtpTraceRestartDelay d ->
+            "ntp client restart delay is "<>toText d
+        NtpTraceRestartingClient ->
+            "ntp client is restarted"
+        NtpTraceClientSleeping ->
+            "ntp client is going to sleep"
+        NtpTraceIOError err ->
+            "ntp client experienced io error " <> toText (show err)
+        NtpTraceLookupServerFailed str ->
+            "ntp client failed to lookup the ntp servers " <> toText str
+        NtpTraceClientStartQuery ->
+            "query to ntp client invoked"
+        NtpTraceNoLocalAddr ->
+            "no local address error when running ntp client"
+        NtpTraceIPv4IPv6NoReplies ->
+            "no replies from servers when running ntp client"
+        NtpTraceReportPolicyQueryFailed ->
+            "policy query error when running ntp client"
```

Comment:
this stuff probably should be put in one module and reused within byron and jormungandr chains

Reply:
added in https://github.com/input-output-hk/cardano-wallet/pull/1396/commits/007aaad5151cccce28237462d8c9d793950f788f